app/services/workflow/plum_workflow_strategy.rb

```diff
@@ -19,7 +20,11 @@ def work_class
       end
 
       def book_works
-        %w(ScannedResource MultiVolumeWork EphemeraBox EphemeraFolder)
+        %w(ScannedResource MultiVolumeWork EphemeraBox)
+      end
+
+      def folder_works
```

Comment:
I named it this because Boxes will probably have a different workflow (due to shipping)